bus_presence_override/static/src/js/systray.js

```diff
@@ -0,0 +1,59 @@
+/* Copyright 2017 LasLabs Inc.
+   License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl). */
+
+odoo.define('bus_presence_override.systray', function (require) {
+    "use strict";
+
+    var DataModel = require('web.DataModel');
+    var session = require('web.session');
+    var SystrayMenu = require('web.SystrayMenu');
+    var Widget = require('web.Widget');
+
+    var Systray = Widget.extend({
+        template:'systray',
+        events: {
+            'click .o_user_presence_status': 'on_click_user_presence_status',
+        },
+        init: function() {
+            this._super.apply(this, arguments);
+            this.Partners = new DataModel('res.partner');
```

Comment:
This naming should be more obvious. As it stands, this alludes to there being partners in this variable, not it being a data model. `this.partnerModel` or something